tests/conftest.py

```diff
@@ -1,29 +1,34 @@
 import json
 from datetime import datetime
 import pytest
+import uuid
 
 from dataservice import create_app
 from dataservice.extensions import db
 from dataservice.api.investigator.models import Investigator
+from dataservice.api.aliquot.models import Aliquot
 from dataservice.api.study.models import Study
 from dataservice.api.participant.models import Participant
 from dataservice.api.family.models import Family
 from dataservice.api.family_relationship.models import FamilyRelationship
 from dataservice.api.diagnosis.models import Diagnosis
 from dataservice.api.sample.models import Sample
-from dataservice.api.aliquot.models import Aliquot
 from dataservice.api.outcome.models import Outcome
 from dataservice.api.phenotype.models import Phenotype
+from dataservice.api.genomic_file.models import GenomicFile
 from dataservice.api.sequencing_experiment.models import SequencingExperiment
 from dataservice.api.study_file.models import StudyFile
 
 
+pytest_plugins = ['tests.mocks']
+
+
 @pytest.yield_fixture(scope='session')
 def app():
     yield create_app('testing')
 
 
-@pytest.yield_fixture(scope='module')
+@pytest.yield_fixture(scope='function')
```

Comment:
Why did we changed the fixture scope to `function` here? is this because app_context will be changes when we access indexd?

Reply:
It's because some of the tests change the state of the db, so it needs to be rebuilt for every test.